tests/cli/workflow/balsamic/test_store_housekeeper.py

```diff
@@ -66,7 +66,7 @@ def test_without_config(cli_runner: CliRunner, balsamic_context: CGConfig, caplo
 
 
 def test_case_without_deliverables_file(
-    cli_runner: CliRunner, balsamic_context: CGConfig, mock_config: dict, caplog
+    cli_runner: CliRunner, balsamic_context: CGConfig, mock_config, caplog
```

Comment:
Was the type hint wrong or could we also remove the parameter?

Reply:
Yes, the type hint was wrong. The fixture does not return anything and does some setup, the test fails without it. I created an issue for it: https://github.com/Clinical-Genomics/cg/issues/2063